Libraries/ActionSheetIOS/RCTActionSheetManager.m

```diff
@@ -168,14 +168,10 @@ - (void)presentViewController:(UIViewController *)alertController
       eventLocationRelativeToWindow = eventLocationInWindow;
     }
     origin = [view convertPoint:eventLocationRelativeToWindow fromView:nil];
-  } else if (view) {
-    CGRect superviewFrame = [superview frame];
-    origin = NSMakePoint(NSMidX(superviewFrame), NSMidY(superviewFrame));
-  } else {
-    origin = [NSEvent mouseLocation];
+    [menu popUpMenuPositioningItem:menu.itemArray.firstObject atLocation:origin inView:superview];
+  } else { // Fallback to mouse location relative to window
+    [menu popUpMenuPositioningItem:menu.itemArray.firstObject atLocation:[NSEvent mouseLocation] inView:nil];
```

Comment:
For code cleanliness/avoiding trivial duplication, would still vote for keeping this duplicate part factored out. Just make this block say view = nil; origin = [NSEvent mouseLocation] and then outside the loop we can call [menu popUpMenuPositioningItem:menu.itemArray.firstObject: atLocation:origin inView:view]